README.md

```diff
@@ -1,8 +1,16 @@
-# catalogue
+# Catalogue
 
 [![Build Status](https://travis-ci.org/wellcometrust/catalogue.svg?branch=master)](https://travis-ci.org/wellcometrust/catalogue)
 
-The Catalogue API allows you to search our museum and library collections.
+
+## Purpose
+
+Making the Wellcome Collection's data and information open and discoverable via public APIs.
+
+You can [read more about how we do this in out documentation][catalogue docs], or just peruse the code at your leisure.
```

Comment:
nit: “our documentation”

If we're going to be linking to docs here, I think we should be a bit clearer about what these docs are, and mention the other type of docs you might find:

* If you want to get data from the API, you can read our docs for users
* If you want to find out how the API works, you can read our docs for platform developers

Exact phrasing could be improved, but part of the problem is that the word “docs” is heavily overloaded and could mean at least half a dozen different things!

Reply:
Maybe even an example of the API? Say:

```
GET https://api.wellcomecollection.org/catalogue/v2/works?query=dinosaur
{
  "@context": "https://api.wellcomecollection.org/catalogue/v2/context.json",
  "type": "ResultList",
  "pageSize": 10,
  "totalPages": 4,
  "totalResults": 33,
  "results": [
    {
      "id": "nsh8dzky",
      "title": "Sex and the dinosaur",
      "alternativeTitles": [],
      "physicalDescription": "v, 205 pages ; 24 cm",
      "workType": {
        "id": "a",
        "label": "Books",
        "type": "WorkType"
      },
      "language": {
        "id": "eng",
        "label": "English",
        "type": "Language"
      },
      "type": "Work"
    },
    ...
}
```

So people have some idea of what the catalogue is about if they've just been linked to this repo.

Reply:
> Maybe even an example of the API? Say:

That can live in developers.weco.org.